builder/frameworks/arduino.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2014-present Ivan Kravets <me@ikravets.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Arduino
+
+Arduino Wiring-based Framework allows writing cross-platform software to
+control devices attached to a wide range of Arduino boards to create all
+kinds of creative coding, interactive objects, spaces or physical experiences.
+
+http://www.stm32duino.com
+"""
+
+from os import walk
+from os.path import isdir, isfile, join
+
+from SCons.Script import DefaultEnvironment
+
+env = DefaultEnvironment()
+platform = env.PioPlatform()
+
+if "stm32f103" in env.BoardConfig().get("build.mcu", ""):
+    FRAMEWORK_DIR = join(platform.get_package_dir("framework-stm32duino"), "STM32F1")
```

Comment:
Please replace text in the all files from `framework-stm32duino` to `framework-arduinoststm32`